CIScripts/BuildFunctions.ps1

```diff
@@ -0,0 +1,151 @@
+class Repo {
+    [string] $Url;
+    [string] $Branch;
+    [string] $DefaultBranch;
+    [string] $Dir;
+
+    Repo ([string] $Url, [string] $Branch, [string] $Dir, [string] $DefaultBranch) {
+        $this.Url = $Url
+        $this.Branch = $Branch
+        $this.Dir = $Dir
+        $this.DefaultBranch = $DefaultBranch
+    }
+}
+
+function Clone-Repos {
```

Comment:
Please use approved words in function names (https://msdn.microsoft.com/en-us/library/ms714428(v=vs.85).aspx)